scripts/validator/.env

```diff
@@ -5,4 +5,5 @@ CONTROLLER_ACCOUNT_JSON=CONTROLLER_ACCOUNT_JSON
 CONTROLLER_ACCOUNT_PASSWORD=CONTROLLER_ACCOUNT_PASSWORD
 BOND_VALUE=999900000000000000
 REWARD_COMMISSION=5
+COMM_MUL=10000000
```

Comment:
It is not going to be changed. No sense to put it in env file. Move it back to code.